ETL/dimAge/LoadDimAge.sql

```diff
@@ -0,0 +1,9 @@
+
+-- etl/LoadDimAge.sql
+LOAD DATA LOCAL INFILE '/Users/marcelacardenas/Desktop/Age.csv'
```

Comment:
I'd recommend uploading this file into a separate folder, something like  `~/data/Age.csv` and do this for all of the other files